### PR TITLE
Python: API to expose highest-level enry.GetLanguage

### PR DESCRIPTION
**Pull request by bzz**
This is a blueprint for all other methods, dealing with go slice conversion.

It still lacks on build automation (and there is no release automation whatsoever), but this is already useful.

### CODE REVIEW

**Comment by bzz** [comment]
CI profiles \w oniguruma have unrelated failure on `Install libonig5`

File: python/enry.py [hunk 1]
```diff
@@ -4,73 +4,86 @@
...
     return go_bool == 1
 
 
-## API
-
+## API, TODO(bzz): add docstrings
+def language(filename: str, content: bytes) -> str:
+    fName, c_str = py_str_to_go(filename)
+    fContent, c_bytes = py_bytes_to_go(content)
+    # print("{} - {}".format(fName, fContent))
```
Comment (lafriks) [inline python/enry.py:33]: this should be removed

Comment (bzz) [reply python/enry.py]: addressed and force-pushed

**Comment by bzz** [comment]
Oniguruma test profiles still fail with (unrelated) 🤔 

```
Run wget "http://archive.ubuntu.com/ubuntu/pool/universe/libo/libonig/libonig5_${ONIGURUMA_VERSION}-1_amd64.deb"
--2020-04-23 14:50:04--  http://archive.ubuntu.com/ubuntu/pool/universe/libo/libonig/libonig5_6.9.1-1_amd64.deb
Resolving archive.ubuntu.com (archive.ubuntu.com)... 91.189.88.152, 91.189.88.142, 2001:67c:1360:8001::23, ...
Connecting to archive.ubuntu.com (archive.ubuntu.com)|91.189.88.152|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
```

**Comment by mcuadros** [comment]
The URL of the package is broken, the same error happened in my local machine:

```
>wget http://archive.ubuntu.com/ubuntu/pool/universe/libo/libonig/libonig5_6.9.1-1_amd64.deb
--2020-04-23 19:19:59--  http://archive.ubuntu.com/ubuntu/pool/universe/libo/libonig/libonig5_6.9.1-1_amd64.deb
Resolving archive.ubuntu.com (archive.ubuntu.com)... 91.189.88.152, 91.189.88.142, 2001:67c:1562::18, ...
Connecting to archive.ubuntu.com (archive.ubuntu.com)|91.189.88.152|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2020-04-23 19:20:00 ERROR 404: Not Found.

```

**Comment by bzz** [comment]
> The URL of the package is broken, the same error happened in my local machine:

@mcuadros thanks for fixing it in #15 🚀 
rebased on latest master and merging.

**Pull request merged**